src/js/poi.js

```diff
@@ -19,7 +19,7 @@ export function missingIconLoader(map, e) {
   var height = sourceSprite.data.height;
 
   var ctx = Gfx.getGfxContext({ width, height });
-  Gfx.copyImageData(ctx, sourceSprite, 0, false, color);
+  Gfx.copyImageData(ctx, sourceSprite, 0, false, color, null);
```

Comment:
Is this needed?

Reply:
Not really, just left it as a reminder to add POI halo coloring later. We can remove it though.